h2o-genmodel/src/main/java/hex/genmodel/easy/EigenEncoderDomainMapConstructor.java

```diff
@@ -0,0 +1,40 @@
+package hex.genmodel.easy;
+
+import hex.genmodel.GenModel;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EigenEncoderDomainMapConstructor extends DomainMapConstructor {
+  
+  public EigenEncoderDomainMapConstructor(GenModel m, Map<String, Integer> columnNameToIndex) {
+    super(m, columnNameToIndex);
+  }
+
+  @Override
+  public Map<Integer, CategoricalEncoder> create() {
+    Map<Integer, CategoricalEncoder> domainMap = new HashMap<>();
+    String[] columnNames = _m.getOrigNames();
+    int pos = 0;
+    for (int i = 0; i < _m.getOrigNumCols(); i++) {
+      String colName = columnNames[i];
+      Integer colIndex = _columnNameToIndex.get(colName);
+      String[] domainValues = _m.getOrigDomainValues()[i];
+      if (domainValues != null) {
+        double[] targetProjectionArray = new double[domainValues.length];
+        System.arraycopy(_m.getOrigProjectionArray() , pos, targetProjectionArray, 0, domainValues.length);
+        pos += domainValues.length;
+        domainMap.put(colIndex, new EigenEncoder(colName, colIndex, domainValues, targetProjectionArray));
+      }
+    }
+    return domainMap;
+  }
+
+  public static long sum(long[] array) {
```

Comment:
use ArrayUtils.sum

Reply:
hm this is not needed at all, it is just some leftover, thx, will remove